.NET/Microsoft.Recognizers.Text.Number/Korean/Extractors/CardinalExtractor.cs

```diff
@@ -1,19 +1,20 @@
 ﻿using System.Collections.Immutable;
 using System.Text.RegularExpressions;
+using Microsoft.Recognizers.Text.Number.Config;
 
 namespace Microsoft.Recognizers.Text.Number.Korean
 {
     public class CardinalExtractor : BaseNumberExtractor
     {
         // CardinalExtractor = Int + Double
-        public CardinalExtractor(KoreanNumberExtractorMode mode = KoreanNumberExtractorMode.Default)
+        public CardinalExtractor(BaseNumberOptionsConfiguration config, CJKNumberExtractorMode mode = CJKNumberExtractorMode.Default)
```

Comment:
BaseNumberOptionsConfiguration already contains mode. So is should be passed as a single parameter.

Reply:
I see, then also Chinese is wrongly implemented, I will update both languages.

Reply:
but they seem to be different modes: 
the one in BaseNumberOptionsConfiguration is a NumberMode with values [Default, Currency, PureNumber, Unit]
the other one is a CJKNumberExtractorMode with values [Default, ExtractAll]

Reply:
I was thinking of merging CJKNumberExtractorMode with NumberMode, but it seems that both are needed separately (NumberExtractor in Chinese/Korean configuration refers to both in different context). 
So not taking actions for now.

Reply:
Got it. I'll take a closer look later.